proto/Makefile

```diff
@@ -16,17 +16,15 @@
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 # Author: Andrew Bonventre (andybons@gmail.com)
 
-PWD=$(shell pwd)
-
-PROTOC      := $(PWD)/../../../../../bin/protoc
+PROTOC      := $(GOPATH)/bin/protoc
```

Comment:
:crying_cat_face: 

we also have similar problems in godeps.sh - is there a reasonable way to handle this?